Airports/EGKK/SMR/Regions.txt

```diff
@@ -845,7 +856,7 @@ N051.09.15.517 W000.10.28.749
 N051.09.12.833 W000.10.27.728
 ;P_Island6
 REGIONNAME Gatwick
-smrGrass N051.09.14.208 W000.10.22.521
+smrGrass 43520 N051.09.14.208 W000.10.22.521
```

Comment:
Just curious as I don't see this '43520' mentioned anywhere else. What is the significance / should this be in?

Reply:
Must have missed this ill remove it when i can be bothered ahha